WalletWasabi.Tests/RegTests.cs

```diff
@@ -1112,6 +1170,18 @@ private void WalletTestsAsync_MemPoolService_TransactionReceived(object sender,
 
 			// allowedInputs cannot be empty
 			await Assert.ThrowsAsync<ArgumentException>(async () => await wallet.BuildTransactionAsync(null, validOperationList, 2, false, null, null, new TxoRef[0]));
+			
+			// "Only one element can contain Money.Zero
+			var toSendWithTwoZeros = new[]{
+				new WalletService.Operation(scp, Money.Zero, "zero"),
+				new WalletService.Operation(scp, Money.Zero, "zero")};
+			await Assert.ThrowsAsync<ArgumentException>(async () => await wallet.BuildTransactionAsync("password", toSendWithTwoZeros, 1008, false));
+
```

Comment:
Can you replace `"password"` with `password`? This was one of my recent changes, so the default password will be initialized in the beginning into a variable.